src/zenml/stack/flavor.py

```diff
@@ -43,6 +43,15 @@ def docs_url(self) -> Optional[str]:
         """
         return None
 
+    @property
+    def sdk_docs_url(self) -> Optional[str]:
+        """A url to point at SDK docs explaining this flavor.
+
+        Returns:
+            A flavor SDK docs url.
+        """
+        return None
```

Comment:
Why dont we just do `return self.generate_default_sdk_docs_url()` here?

Reply:
I was basing what I did here on what was implemented for the `docs_url`. Presumed that there was maybe something about the frontend that required this? I'll check with @AlexejPenner when he's back on whether that's the case. Otherwise, I can update it for both here and just return a string and not an optional string.

Reply:
Same as above, this breaks custom user flavors. I guess you could implement some check here to see if this is a zenml flavor and only then return it? However it will soon change with the hub anyway so I wouldn't spend too much time on it,